services/platform/platform.go

```diff
@@ -356,6 +362,10 @@ func (s *Service) SearchOrganizationTickets(ctx context.Context, req *platformpb
 		return nil, errGetSSODetailsFailed
 	}
 
+	if ssoDetails.OrganizationID == "" {
+		return nil, errOrgIDNotFound
+	}
```

Comment:
can we get this value from portal?

Reply:
i realized we no longer need this condition since orgID is now not null so I removed it.

Reply:
it still can be empty value, but ok